dbt-tests-adapter/dbt/tests/adapter/basic/test_incremental.py

```diff
@@ -86,9 +86,37 @@ def test_incremental_not_schema_change(self, project):
         assert run_result == RunStatus.Success
 
 
+class BaseIncrementalBadStrategy:
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {"name": "incremental"}
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "incremental.sql": files.incremental_invalid_strategy_sql,
+        }
+
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {"base.csv": files.seeds_base_csv, "added.csv": files.seeds_added_csv}
+
+    def test_incremental_invalid_strategy(self, project):
+        # seed command
+        results = run_dbt(["seed"])
+        assert len(results) == 2
+
+        # try to run the incremental model, it should fail on the first attempt
+        results = run_dbt(["run"], expect_pass=False)
+
+
 class Testincremental(BaseIncremental):
     pass
 
 
 class TestBaseIncrementalNotSchemaChange(BaseIncrementalNotSchemaChange):
     pass
+
+
+class TestBaseIncrementalBadStrategy(BaseIncrementalBadStrategy):
```

Comment:
We don't need this anymore. This used to be here because this package used to also serve as the integration tests for `dbt-postgres`, which is now it's own repo.